### PR TITLE
Feature/reset password modal

### PR DESCRIPTION
**Pull request by dmytro-chushko**
Add modal for expired link.
 - recover-pass-second-formHooks.ts
 - recover-pass-seconfd-form.tsx
 - image

### CODE REVIEW

File: libs/components/src/lib/recover-pass-second-form/recover-pass-second-formHooks.ts [hunk 1]
```diff
@@ -7,24 +8,35 @@ import { toast } from "react-toastify";
 import "react-toastify/dist/ReactToastify.css";
 
 const PASSWORD_UPDATED = "/password-updated";
+const RECOVER_PASSWORD = "/recover-password";
 
 export function useResetPassword(): IonSubmitResetPassword {
 	const { t } = useTranslation();
+	const [open, setModalOpen] = useState(false);
```
Comment (lyudad) [inline libs/components/src/lib/recover-pass-second-form/recover-pass-second-formHooks.ts]: types

File: libs/components/src/lib/recover-pass-second-form/recover-pass-second-formHooks.ts [hunk 2]
```diff
@@ -7,24 +8,35 @@ import { toast } from "react-toastify";
...
 	const navigate = useNavigate();
 	const { token } = useParams();
 	const [resetPassword, { isLoading }] = useResetPasswordMutation();
 
+	const handleModalOk = () => {
+		navigate(RECOVER_PASSWORD);
+	};
```
Comment (lyudad) [inline libs/components/src/lib/recover-pass-second-form/recover-pass-second-formHooks.ts:22]: Снимок экрана 2022-11-20 в 19 58 52
все еще не настроен еслинт

File: libs/components/src/lib/recover-pass-second-form/recover-pass-second-formHooks.ts (hunk 1 above)
Comment (dmytro-chushko) [reply libs/components/src/lib/recover-pass-second-form/recover-pass-second-formHooks.ts]: added types

File: libs/components/src/lib/recover-pass-second-form/recover-pass-second-formHooks.ts (hunk 2 above)
Comment (dmytro-chushko) [reply libs/components/src/lib/recover-pass-second-form/recover-pass-second-formHooks.ts:22]: added eslint configurations

**Review by lyudad — APPROVED** [review]

**Pull request merged**